hydpy/docs/rst/install.rst

```diff
@@ -49,7 +49,7 @@ installer.
 Selecting a Python distribution
 --------------------------------
 
-Using *HydPy* requires installing `Python 3.7, 3.8, 3.9, or 3.10`_ first.  You
+Using *HydPy* requires installing `Python 3.8, 3.9, 3.10 or 3.11`_ first.  You
```

Comment:
Travis says: 

Warning, treated as error:
/home/travis/build/hydpy-dev/hydpy/.nox/sphinx/lib/python3.8/site-packages/hydpy/docs/auto/install.rst:53:Unknown target name: "python 3.8, 3.9, 3.10 or 3.11".

The target is defined above in the same file.

You can use the nox session `sphinx` to test if documentation changes do not break any references.